### PR TITLE
rely on `FacilityFeature.Providers`

### PR DESCRIPTION
**Pull request by cainlevy**
depends on https://github.com/Ameelio/connect-api-nest/pull/1163

### CODE REVIEW

File: src/Organizations/GetPartnerOrganizations.graphql [hunk 1]
```diff
@@ -3,6 +3,7 @@ query GetPartnerOrganizations(
   $requestStatus: FacilityOrganizationStatus
 ) {
   facility(id: $facilityId) {
+    id
```
Comment (cainlevy) [inline src/Organizations/GetPartnerOrganizations.graphql:6]: unrelated fix for an apollo cache warning

File: src/components/Menu/Menu.tsx [hunk 2]
```diff
@@ -294,48 +293,44 @@ export default function Menu({
           )}
         </AntdMenu.SubMenu>
 
-        {(featureEnabled("PROVIDERS") || isSuperUser(user)) && (
```
Comment (cainlevy) [inline src/components/Menu/Menu.tsx:297]: we decided to show the pages in our navigation but then display a placeholder if someone clicks through.

i imagine the placeholder will be improved. it's very placeholder-y right now.

File: src/pages/OrganizationsPage.tsx [hunk 3]
```diff
@@ -124,7 +127,7 @@ export default function OrganizationsPage(): ReactElement {
         ]}
       />
       <div style={WRAPPER_STYLE}>
-        <Table dataSource={facilityOrgs} loading={loading}>
+        <Table dataSource={facilityOrgs} loading={loading} rowKey="id">
```
Comment (cainlevy) [inline src/pages/OrganizationsPage.tsx:130]: unrelated fix for a key warning

**Review by dabbler0 — APPROVED** [review]
